### PR TITLE
Windows update: Don't kill keybase until --kill-kbfs is issued

### PR DESCRIPTION
**Pull request by zanderz**
This fixed the Win7 32 silent update case I found wasn't working today - but note that, unfortunately, those users will have to reboot still because this change is not in the (un)installer they currently have.
@maxtaco @cjb @oconnor663

### CODE REVIEW

File: packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs [hunk 1]
```diff
@@ -30,8 +30,8 @@
     <InstallExecuteSequence>
       <InstallValidate Suppress="yes">FAKE_PROPERTY</InstallValidate>
       <Custom Action="StopUpdater" Before="InstallValidate"/>
+      <Custom Action="StopMainApp" Before="StopGUI"/>
```
Comment (maxtaco) [inline packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs]: There's no scenario in which the GUI will try to restart the main app, is there?

Comment (zanderz) [inline packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs]: It can do that? I didn't see it today.

Comment (maxtaco) [inline packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs]: In some cases on OSX, the GUI can start the App.  I'm guessing not here, but thought I'd double check

**Comment by songgao** [comment]
@zanderz I merged master back to your branch. There was an API change yesterday which is why the CI is failing I think

**Comment by maxtaco** [comment]
👍

**Pull request merged**